.gitignore

```diff
@@ -24,3 +24,164 @@ QtWaitingSpinner.exe
 build-QtWaitingSpinnerTest-Desktop-Debug*
 build-QtWaitingSpinnerTest-GCC-Debug*
 build-QtWaitingSpinnerTest-Desktop_Qt_5_3_GCC_64bit-Debug*
+
```

Comment:
I've added the contents of GitHub's .gitignore template for Python projects, mostly because I almost accidentally committed the `__pycache__` folder or other files a couple of times. 
It makes sense to me that this wasn't included earlier because this project was originally forked from a C++ project (https://github.com/snowwlex/QtWaitingSpinner), but I think this will make things a lot more convenient.